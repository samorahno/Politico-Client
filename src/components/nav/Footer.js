import React from 'react';
import '../../styles/landing.scss';

const Footer = () => {
    return (
        <div>
            <footer className="steps footercolor">
                <div className="column step">
                    <table className="footer-table">
                        <tbody>
                            <tr>
                                <td><span className="footer-td">
                                    <p><i className="fa fa-phone" /> : +2348060955366 <br /> <i className="fa fa-map-marker" /> : 28,
                                        Adewale Street, Victoria Island, Lagos, Nigeria <br /><i className="fa fa-envelope" /> :
                                        samabos007@gmail.com
                                    </p>
                                </span>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                </div>
                <div className="column step">
                    <p>
                        Politico | by Abosede Samson &copy; 2019
                    </p>
                    <p />
                </div>
            </footer>
        </div>
    );
};

export default Footer;
