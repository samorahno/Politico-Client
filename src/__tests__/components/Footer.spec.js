import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/nav/Footer';

describe('Footer', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Footer debug/>);
        expect(component).toMatchSnapshot();
    });
});
