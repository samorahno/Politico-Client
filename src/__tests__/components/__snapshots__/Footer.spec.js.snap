// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
